apps/frontend/src/components/global/upload_tabs/tenable/TenableReader.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <v-stepper v-model="step">
+    <v-stepper-header class="elevation-0">
+      <v-stepper-step id="step-1" step="1">
+        Tenable Authorization
+      </v-stepper-step>
+      <v-divider />
+      <v-stepper-step id="step-2" step="2">
+        Search Scan Results
+      </v-stepper-step>
+    </v-stepper-header>
+    <v-stepper-items>
+      <v-stepper-content step="1">
+        <AuthStep
+          @authenticated="onAuthenticationComplete"
+          @error="errorCount += 1"
+          @show-help="errorCount = -1"
+        />
+      </v-stepper-content>
+      <v-stepper-content step="2">
+        <FileList
+          v-if="tenableConfig"
+          :tenable-config="tenableConfig"
+          @signOut="onSignOut"
+          @got-files="got_files"
+        />
+      </v-stepper-content>
+    </v-stepper-items>
+    <v-overlay
+      :opacity="50"
+      absolute="absolute"
+      :value="errorCount >= 3 || errorCount < 0"
```

Comment:
why is it possible for the errorcount to be less than 0?

why are we checking for if it's greater than or equal to 3 here, but on line 36 for the v-if on that span we only check if it's greater than 0?

Reply:
so we can display the error span tag